src/wp-includes/class-wp-theme.php

```diff
@@ -1840,6 +1798,213 @@ public function get_block_template_folders() {
 		return $this->block_template_folders;
 	}
 
+	/**
+	 * Gets block pattern data for a specified theme.
+	 * Each pattern is defined as a PHP file and defines
+	 *  its metadata using plugin-style headers. The minimum required definition is:
+	 *
+	 *      /**
+	 *       * Title: My Pattern
+	 *       * Slug: my-theme/my-pattern
+	 *       *
+	 *
+	 *  The output of the PHP source corresponds to the content of the pattern, e.g.:
+	 *
+	 *      <main><p><?php echo "Hello"; ?></p></main>
+	 *
+	 *  If applicable, this will collect from both parent and child theme.
+	 *
+	 *  Other settable fields include:
+	 *
+	 *    - Description
+	 *    - Viewport Width
+	 *    - Inserter         (yes/no)
+	 *    - Categories       (comma-separated values)
+	 *    - Keywords         (comma-separated values)
+	 *    - Block Types      (comma-separated values)
+	 *    - Post Types       (comma-separated values)
+	 *    - Template Types   (comma-separated values)
+	 *
+	 * @since 6.4.0
+
+	 * @return array Block pattern data.
```

Comment:
Fix missing docblock line.

```suggestion
	 * @since 6.4.0
	 *	 
	 * @return array Block pattern data.
```